data/modules/Scout/Scout.lua

```diff
@@ -568,14 +568,14 @@ end
 local onUpdateBB = function (station)
 	for ref,ad in pairs(ads) do
 		if not flavours[ad.flavour].localscout
-			and ad.due < Game.time + 432000 then -- 5 days
+			and ad.due < Game.time then -- 5 days
 			ad.station:RemoveAdvert(ref)
 		elseif flavours[ad.flavour].localscout
-			and ad.due < Game.time + 172800 then -- 2 days
+			and ad.due < Game.time then -- 2 days
```

Comment:
isn't the comment wrong now?  172800 was the 2 days? Same on L571 with 5 days

Reply:
Thanks, fixed!